prqlc/bindings/dotnet/.editorconfig

```diff
@@ -0,0 +1,175 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+root = true
+
+# Default settings:
+# A newline ending every file
+# Use 4 spaces as indentation
+[*]
+insert_final_newline = true
+indent_style = space
+indent_size = 4
+trim_trailing_whitespace = true
+dotnet_style_operator_placement_when_wrapping = beginning_of_line
+tab_width = 4
+end_of_line = crlf
```

Comment:
Why is this?

Reply:
If you've another .editorconfig to propose feel free to put it.

Reply:
I don't have an opinion on Charp-specific settings, I just think it is safer to basically enforce LF since we once had a problem with different line break codes in this repository.
https://github.com/PRQL/prql/blob/56bdbe4bd23ca3c2b625c0559d01ecdde078ecdf/.gitattributes#L6

Reply:
In other words, there is a discrepancy between the settings in Git and Editorconfig.
I believe that Git creates the files wish LF, but when Editorconfig is run, it is immediately rewritten in CRLF.
This means that the newline code changes depending on whether or not Editorconfig is run, which I believe causes trouble.

Reply:
Ok make sense. Done.